deploy/stacks/lambda_api.py

```diff
@@ -225,6 +227,7 @@ def __init__(
                 'custom_auth_url': custom_auth.get('url'),
                 'custom_auth_client': custom_auth.get('client_id'),
                 'custom_auth_jwks_url': custom_auth.get('jwks_url'),
+                'ALLOWED_ORIGINS': allowed_origins,
```

Comment:
don't think we will need to set ALLOWED_ORIGINS for custom authorizer lambda... this lambda is for custom auth i not using cognito IdP and does not handle API requests

Reply:
Removed this as well.